.github/workflows/release.yml

```diff
@@ -49,6 +49,10 @@ jobs:
           distribution: corretto
           java-version: 11
           cache: sbt
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: .nvmrc
+          registry-url: https://registry.npmjs.org
       - name: Release Production to Sonatype
```

Comment:
Minor gripe - it's now Sonatype _and_ npm.  The amount of stuff that the sonatype release spews to console it's going to be be difficult to tell!

Reply:
ea824e10c14c5d402f9ff79e5416be2ddbb934a6